invenio_rdm_records/resources/serializers/bibtex/schema.py

```diff
@@ -0,0 +1,194 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""BibTex based Schema for Invenio RDM Records."""
+
+import datetime
+
+import six
+from marshmallow import Schema, fields, missing, post_dump
+from slugify import slugify
+
+from ..schemas import BaseSchema
+from .schema_formats import BibTexFormatter
+
+
+class BibTexSchema(BaseSchema):
+    """Schema for records in BibTex."""
+
+    id = fields.Method("get_id")
```

Comment:
In this case, I'm not sure it needs a method field. Would it work by putting a SanitizedUnicode or Str (check for example what is used for id in the main `service/schema/`?

```suggestion
    id = fields.Str()
```